user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+
+    
+
+Adjust a Grid
+-------------
+
+.. _user-guide-ui-components-grids-change-table:
+
+Change the Table
+^^^^^^^^^^^^^^^^
+
+Actually, you can change the set of details available in the grid (if permitted), and the order of columns.
+To do so, go to the right corner of the grid, and click the settings icon |IcSettings|.
+
+To show/hide a column on the grid, check/uncheck it in the *"Table Settings"*.(The list of columns is defined 
+by the system administrator with the  *Show On Grid* setting of the entity fields).
+
+To move columns of the grid, move them in the form: click on the "Sort" column and hold the mouse button - drag the 
+column to change their order:
+
+|
+
+.. image:: ./img/data_management/grid/grid_table_settings.png
+
+|
+
+.. _user-guide-ui-components-grid-filters:
+
+Apply Filters
+^^^^^^^^^^^^^
+
+You can apply filters to choose specific items to be shown in the grid. 
+
+Of course, if there are a lot of contacts, they all won't fit to one page. In order to find the required contact in the 
+grid, you can use the grid filters. 
+
+Some filters are available by default, but the set of filters can be modified with check-boxes under the 
+*"Manage Filters"* menu on the left. (The list of filters is defined 
+by the system administrator with the *Show Grid Filter* setting of the entity fields).
+
+|
+
+.. image:: ./img/data_management/grid/grid_filters.png
+
+|
+
+The list of properties that can be used as filters is defined by the system administrator with *"Show Grid Filter"* 
+setting of the entity fields. 
+
+
+Click the arrow on a filter to choose required values. Available value setting depend on the field type. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_filters_define.png
+
+|
+
+If more than one filter is active, only the records that meet requirements of ALL the filters are displayed. For 
+example, we have filtered the *"All Contacts"* grid so that it would only show Jameses Browns from California.
+
+.. image:: ./img/data_management/grid/grid_02.png
+
+
+Change Sorting
+^^^^^^^^^^^^^^
+
+Be default the grids are sorted by the first column in the ascending order. You can change the sorting by any field and 
+in any order - it is enough to click the column header. An upward or downward looking arrow will appear by the column 
+name.
+
+
+.. _user-guide-ui-components-grid-customized:
+
+Save Your Customized Grid View and Share It with Other Users
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If there is a frequent set of filters and/or ordering that you need to use, you can save your 
+customized grid view with all the filters and return to 
+it at any moment. You can have any number of such views. This is very convenient, for example, if 
+you are working with the customers from different stores, or contacts from different states, and so on.
+
+
+You can save a set of filters and sorting applied to the grid in a named view.
```

Comment:
Edit: You can save a set of filters and any custom sorting you've applied to the grid using the named view.